reactive/kubernetes_master.py

```diff
@@ -2059,7 +2081,10 @@ def configure_apiserver():
     else:
         remove_if_exists(audit_webhook_config_path)
 
-    if is_flag_set('kubernetes-master.secure-storage.created'):
+    if kube_version > (1, 8):
```

Comment:
I wouldn't even bother with this check personally. Charmed Kubernetes 1.19 does not need to support anything earlier than Kubernetes 1.17.